### PR TITLE
Gwells-2211 *FEATURE add closed/abandoned wells colour

### PR DESCRIPTION
**Pull request by lunamoonmoon**
## Pull Request Standards

- [x] The title of the PR is accurate
- [x] The title includes the type of change [`HOTFIX`, `FEATURE`, `etc`]  
- [x] The PR title includes the ticket number in format of `[GWELLS-###]`
- [x] Documentation is updated to reflect change [`README`, `functions`, `team documents`]

This PR includes the following proposed change(s):

- Added case to make wells that have well_status as closure or abandoned a deep purple colour.
- Added case to make wells that have well_status as closure or abandoned have a transparent outline and not take on the artesian fushcia outline.
- Added a pulsing dot image in deep purple with lighter pulse colour to match pulsing artesian and pulsing well images.
- Added to legend as decommissioned wells with deep purple dot.
- ~~Noticed that dot outline 'circle stroke color' was transparent default on load but black on search, corrected both to black outline.~~ *reverted this change after discussion with devs

- Screenshot 2024-07-05 at 1 47 27 PM

~~These shows the current discrepancy between well outlines (transparent vs black) on the main page vs search page~~
- Screenshot 2024-07-05 at 1 45 35 PM
Screenshot 2024-07-05 at 1 46 16 PM

### CODE REVIEW

**Review by dallascrichmond — APPROVED** [review]
Looks good :)

File: app/frontend/src/common/mapbox/layers.js [hunk 1]
```diff
@@ -278,12 +278,16 @@ export function wellsBaseAndArtesianLayer (options = {}) {
...
+      ['==', ['get', 'well_status'], 'ABANDONED'], '#302d57',
       ['to-boolean', ['get', 'artesian']], '#1099FE',
       '#0162FE'
     ],
     'circle-radius': 3,
     'circle-stroke-color': [
       'case',
+      ['==', ['get', 'well_status'], 'CLOSURE'], 'transparent',
+      ['==', ['get', 'well_status'], 'ABANDONED'], 'transparent',
       ['to-boolean', ['get', 'artesian']], '#EE14CA',
```
Comment (acatchpole) [inline app/frontend/src/common/mapbox/layers.js:291]: When you're presenting the changes to the SO, make sure you highlight everything listed in the PR (sometimes the comments don't get read) and specifically this one. Given my understanding of how artesian conditions are used, they might want wells that have them to still be highlighted even when closed. I'm not saying undo the change right now, but double check with the SO.

Comment (lunamoonmoon) [reply app/frontend/src/common/mapbox/layers.js:291]: Ack, I will do that now, thank you!
